java.yaml

```diff
@@ -11,9 +11,13 @@ steps:
   - '*|*=gcr.io/google-appengine/jetty:9'
   - 'openjdk8|*=gcr.io/google-appengine/jetty:9'
   - 'openjdk8|jetty9=gcr.io/google-appengine/jetty:9'
+  - 'openjdk8|jetty=gcr.io/google-appengine/jetty:9'
```

Comment:
Presumably at some point in the future, the jetty:latest point to an image that's based on Java 9. When that's the case, it wouldn't make sense for the user to specify `jdk: openjdk8` and get a java 9 based runtime.

IMO the correct behavior is to select the latest tag of the jetty runtime that uses openjdk8 (which is jetty:9).